README.md

```diff
@@ -44,18 +44,31 @@ sudo npm link
 
 ## Quick Start
 
-Create a `specs` folder in your Titanium app (at the same level as the `Resources` folder).  The spec should be in mocha format.
+Create a `suites` folder in your Titanium app (at the same level as the `Resources` folder), and then
+create multiple sub-folders there, each sub-folders must contain an `app.js` file, like:
 
```

Comment:
You now have a suites folder and a harness folder. Harness folder contains the Titanium app. Do note that the harness app https://github.com/appcelerator/tio2/blob/master/example/harness/Resources/app.js is needed as it contains methods for running the tests.